docs/attributes-registry/thread.md

```diff
@@ -10,6 +10,6 @@ These attributes may be used for any operation to store information about a thre
 <!-- semconv registry.thread(omit_requirement_level) -->
```

Comment:
Unrelated to the change in this PR: do we want to a "Document Status" for docs in the attributes-registry? I see we have them in the other docs but not here.